test/batch_normalization_test.js

```diff
@@ -2,10 +2,10 @@
 
 import {batchNormalization} from '../src/batch_normalization.js';
 import {clamp} from '../src/clamp.js';
-import {leakyRelu} from '../src/leaky_relu.js';
+import {leakyRelu} from '../src/lib/leaky_relu.js';
```

Comment:
`leakyRelu` is a WebNN op: https://www.w3.org/TR/webnn/#api-mlgraphbuilder-leakyrelu